acs-integration-tests/src/test/java/com/ge/predix/integration/test/ACSMeteringIT.java

```diff
@@ -174,6 +191,30 @@ private Double getEntitlementUsageByFeatureId(final String featureId, final Stri
         return entitlement.getCurrentUsedAmount();
     }
 
+    private double getSteadyEntitlementUsage(final String featureId, final String subscriptionId)
+            throws Exception {
+        LOGGER.info("Entered Meter Steady Stage");
+
+        double entitlementUsage_1 = 0.0;
+        double entitlementUsage_2 = 0.0;
+
+        for( int i = 0; i < MAX_STEADY_ITERATIONS; i++ ) {
+            entitlementUsage_1 = getEntitlementUsageByFeatureId(featureId, subscriptionId);
+
+            Thread.sleep(10000);
+
+            entitlementUsage_2 = getEntitlementUsageByFeatureId(featureId, subscriptionId);
+
+            if(entitlementUsage_1 == entitlementUsage_2) {
+                LOGGER.info("Meter has reached a steady state for " + POLICY_UPDATE_FEATURE_ID);
```

Comment:
This log message would be more useful if it it included more information e.g.

```String.format("Meter has reached steady state of '%d' for subscription '%s' and feature '%s'.", entitlementUsage_1, featureId, subscriptionId)```

Also consider making this a debug level message rather than an info message. In general you want logging to be as sparse as possible but when you do log you want it to be as information dense as possible. Nobody likes log messages that are difficult to interpret or that don't contain enough useful information.